libraries/abstractions/common_io/test/test_iot_i2c.c

```diff
@@ -806,7 +807,7 @@ TEST( TEST_IOT_I2C, AFQP_IotI2CWriteAsyncSuccess )
     IotI2CHandle_t xI2CHandle;
     int32_t lRetVal;
     uint8_t writeVal1[] = { xtestIotI2CDeviceRegister, uctestIotI2CWriteVal };
-    uint8_t writeVal2[] = { xtestIotI2CDeviceRegister, uctestIotI2CWriteVal, uctestIotI2CWriteVal, uctestIotI2CWriteVal };
+    uint8_t writeVal2[ IOT_TEST_COMMON_IO_I2C_MAX_WRITE_LENGTH ] = { xtestIotI2CDeviceRegister, uctestIotI2CWriteVal, uctestIotI2CWriteVal };
```

Comment:
Is this change because you want extra space after the initialized portion of the buffer? Is there a comment somewhere that explains why?

Reply:
I made this change because 4 bytes might be too large for some case (TI) 
I probably should write it to below, implying the IOT_TEST_COMMON_IO_I2C_MAX_WRITE_LENGTH >= 2
```
uint8_t writeVal2[ IOT_TEST_COMMON_IO_I2C_MAX_WRITE_LENGTH ] = { xtestIotI2CDeviceRegister, uctestIotI2CWriteVal };
```